server/pension/models.py

```diff
@@ -34,30 +115,28 @@ def __str__(self):
         return '{} / {}'.format(self.quarter, self.year)
 
 
-class ManagingBody(models.Model):
-    """
-    The body that owns the funds.
-    """
+class Fund(models.Model):
     label = models.CharField(max_length=200)
 
     def __str__(self):
         return self.label
 
-
-class Fund(models.Model):
+class Issuer(models.Model):
+    """
+    Company (or other body) that issues a security.
+    """
     label = models.CharField(max_length=200)
 
     def __str__(self):
         return self.label
 
-
 class FundManagingBody(models.Model):
```

Comment:
@ranmoshe Shouldn't delete this blank line. (PEP8)[https://www.python.org/dev/peps/pep-0008/#blank-lines] says 2 blank line before any class in `Python`.

* if you'll use PyCharm it'll mark this as an error.